src/System.Management.Automation/engine/NativeCommandProcessor.cs

```diff
@@ -1171,17 +1171,19 @@ private void CleanUp()
 
             try
             {
-                // on Unix, we need to kill the process to ensure it terminates as Dispose() merely
+                // on Unix, we need to kill the process (if not running in background) to ensure it terminates as Dispose() merely
```

Comment:
Suggestion: add comma to make it clearer, as I initially read it as "terminates as Dispose()" and wondered how can the process terminate as a disposable.

```suggestion
                // on Unix, we need to kill the process (if not running in background) to ensure it terminates, as Dispose() merely
```